src/java/us/kbase/scripts/test/TypeGeneratorTest.java

```diff
@@ -989,30 +988,33 @@ private static File prepareWorkDir(int testNum) throws IOException {
 		return workDir;
 	}
 
-	private static void runClientTest(int testNum, String testPackage, JavaData parsingData, 
-			File libDir, File binDir, int portNum, boolean needClientServer, File outDir,
-			String serverType) throws Exception {
-	    ensureCasperJsInstalled();
-	    System.out.println("- Java client -> " + serverType + " server");
-	    runJavaClientTest(testNum, testPackage, parsingData, libDir, binDir, portNum, needClientServer);
-	    if (outDir != null) {
-            String resourceName = "Test" + testNum + ".config.properties";
-            String clientConfigText = checkForClientConfig(resourceName);
-	        if (clientConfigText == null) {
-	            System.err.println("- Perl/Python/JavaScript client tests are skipped (" + resourceName + "not found)");	
-	            return;
-	        }
-	        if (clientConfigText.isEmpty())
-                return;
-	        System.out.println("- Perl client -> " + serverType + " server");
-	        runPerlClientTest(testNum, testPackage, parsingData, portNum, needClientServer, outDir);
-	        System.out.println("- Python client -> " + serverType + " server");
-            runPythonClientTest(testNum, testPackage, parsingData, portNum, needClientServer, outDir, false);
-            System.out.println("- Python3 client -> " + serverType + " server");
-            runPythonClientTest(testNum, testPackage, parsingData, portNum, needClientServer, outDir, true);
-            System.out.println("- JavaScript client -> " + serverType + " server");
-            runJsClientTest(testNum, testPackage, parsingData, portNum, needClientServer, outDir);
-	    }
+	private static void runClientTest(
+			final int testNum,
```

Comment:
I get that you're just trying to do the minimal to fix things, but there's gotta be a way to initialize tests without a minimum of 10 freaking arguments.

Not to mention the compile statements.

Anyway. Just complaining about existing code. Carrying on.

Reply:
You're preaching to the choir here. Tests passing first, then cleanup